frontend/src/components/AddMember/index.js

```diff
@@ -1,168 +1,243 @@
+    ],
     yesNo: [
-      { key: "true", value: "Da" },
-      { key: "false", value: "Nu" }
+      { value: true, text: "Da" },
+      { value: false, text: "Nu" }
     ],
-    preexistingConditions: [
-      { value: "1", text: "Boală cardiovasculară" },
+    preexistingMedicalCondition: [
+      {
+        value: "1",
+        text: "O boală de inimă (boală cardiovasculară, înclusiv hipertensiune"
+      },
       { value: "2", text: "Diabet" },
-      { value: "3", text: "Boală pulmonară" },
+      { value: "3", text: "O boală a plămânilor" },
       { value: "4", text: "Cancer" },
       { value: "5", text: "Altă boală cronică" }
     ],
-    isolation: [
-      { key: "1", value: "Auto izolare" },
-      { key: "2", value: "Carantină la domiciliu" },
-      { key: "3", value: "Carantină specializată" },
-      { key: "4", value: "Niciuna" }
+    quarantineStatus: [
+      { value: "1", text: "Da, sunt în izolare, nu ies deloc din locuință" },
+      {
+        value: "2",
+        text: "Stau mai mult pe acasă, dar mai ies atunci când este nevoie"
+      },
+      {
+        value: "3",
+        text: "Nu sunt în izolare și ies din locuință după program normal"
+      },
+      {
+        value: "4",
+        text: "Altă situație"
+      }
     ]
   };
 
-  const toggleSex = event => {
-    setFemale(event.target.value === "Feminin");
-  };
-
-  const togglePreexistingConditions = value => {
-    setPreexistingConditions(value === "Da");
-  };
+  const [currentStep, setCurrentStep] = useState(0);
+  const [userData, setUserData] = useState({
+    name: "",
+    surname: "",
+    phoneNumber: "",
+    relationType: 0,
+    age: 0,
+    gender: 0,
+    smoker: false,
+    preexistingMedicalCondition: "",
+    livesWithOthers: false,
+    quarantineStatus: 0,
+    quarantineStatusOther: 0
+  });
 
-  const toggleDisabilities = value => {
-    setDisabilities(value === "Da");
+  const setUserDataField = (field, value) => {
+    setUserData({
+      ...userData,
+      [field]: value
+    });
   };
 
-  const validateAge = event => {
-    const value = parseInt(event.target.value.replace(/[^0-9]+/g, ""));
-    // age range is between 0 and 110 - might be worth reconsidering?
-    const boundedValue = Math.min(110, Math.max(0, value));
+  const personalDataForm = (
+    <>
+      <ListHeader title="I. Date personale"></ListHeader>
+      <div className="user-profile-form-container">
+        <Input
+          type="text"
+          label={"Nume"}
+          usePlaceholder
+          defaultValue={userData.name}
+          onChange={({ currentTarget: { value } }) => {
+            setUserDataField("name", value);
+          }}
+        />
+        <Input
+          type="text"
+          label={"Prenume"}
+          usePlaceholder
+          defaultValue={userData.surname}
+          onChange={({ currentTarget: { value } }) => {
+            setUserDataField("surname", value);
+          }}
+        />
+        <Input
+          type="text"
```

Comment:
We have a [`tel` input type](https://taskforce-fe-components.now.sh/?path=/story/input--types)